enhancements/api-review/custom-tags-aws.md

```diff
@@ -91,19 +151,23 @@ AWS resource tags.
 
 ### Test Plan
 
-TBD
+Update the userTag testcases to check successful update from infrastructure resource.
```

Comment:
```suggestion
Update the `resourceTags` testcases to check successful update from infrastructure resource.
```

Reply:
Capital "I" in all Infrastructure occurrences?